samples/meeting-av-filters/csharp/MeetingAVFilter/ClientApp/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
```

Comment:
Check if we are using all these CSS ?

Reply:
yes the css is being used for default react html page